vignettes/getting-started-with-teal.Rmd

```diff
@@ -36,52 +36,36 @@ library(teal)
 app <- init(
   data = teal_data(IRIS = iris, MTCARS = mtcars),
   modules = example_module(),
-  header = "My first teal application"
+  title = "My first teal application",
```

Comment:
```suggestion
  title = build_app_title(title = "My first teal application"),
```

What do you think about using the `build_app_title` over here?

Reply:
Thanks, You right! I didn't know that this function even exists ;]